keyboard/PlayerKeyboardControl.ts

```diff
@@ -0,0 +1,451 @@
+
+    private keyMap: PlayerKeyMap;
+    private isEnabled: boolean;
+    private player: SupportedPlayerTypes;
+
+    constructor(wrappedPlayer: SupportedPlayerTypes, config?: any) {
+        this.player = wrappedPlayer;
+        let paramKeyMap = {};
+        if (config) {
+            paramKeyMap = config;
+        }
+
+        this.keyMap = PlayerKeyboardControl.mergeConfigWithDefault(paramKeyMap);
+        // default to enabled
+        // this also registers the event listeners
+        this.enable(true);
+
+        // destroy this together with the player
+        this.player.addEventHandler('onDestroy', () => {
+            this.destroy();
+        });
+    }
+
+    public enable(shouldBeEnabled: boolean = true) {
+        this.isEnabled = shouldBeEnabled;
+        // depending if we are enabled register or remove the keyListener
+        // we cannot use the keypress event as that event does not work with modifiers
+        // only add the keyUp listener as we do not expect users holding buttons to control the player
+        if (this.isEnabled) {
+            // document.addEventListener('keypress', this.handleKeyEvent, false);
+            // document.addEventListener('keydown', this.handleKeyEvent, false);
+            document.addEventListener('keyup', this.handleKeyEvent, false);
+        } else {
+            // document.addEventListener('keypress', this.handleKeyEvent, false);
+            // document.removeEventListener('keydown', this.handleKeyEvent, false);
+            document.removeEventListener('keyup', this.handleKeyEvent, false);
+        }
+    }
+
+    public disable(shouldBeDisabled: boolean = true) {
+        this.enable(!shouldBeDisabled);
+    }
+
+    public destroy() {
+        // removes the listener
+        this.disable(true);
+    }
+
+    protected static mergeConfigWithDefault(paramKeyMap: object): PlayerKeyMap {
+        let retVal = new DefaultPlayerKeymap();
+        // allow overwrites to the default player keymap as well as new listeners
+        for (let attr in paramKeyMap) {
+            if (attr && paramKeyMap[attr]) {
+                let toCheck = paramKeyMap[attr];
+                // avoid wrong configs and check for elements being real keyListeners
+                if (toCheck['keyBinding'] && toCheck['callback']) {
+                    retVal[attr] = paramKeyMap[attr];
+                } else {
+                    console.log('Invalid Key Listener at params[' + attr + ']');
+                }
+            }
+        }
+        return retVal;
+    }
+
+    public handleKeyEvent = (event: KeyboardEvent) => {
+        if (this.isEnabled) {
+            let keyStringRepresentation = KeyboardEventMapper.convertKeyboardEventToString(event);
+
+            let bindings = this.keyMap.getAllBindingsForKey(keyStringRepresentation);
+            bindings.forEach((singleBinding: KeyToFunctionBinding) => {
+                singleBinding.callback(this.player);
+            });
```

Comment:
This alone was not enough. `event.preventDefault()` nor `event.stopPropagation()` stopped Chrome or Firefox from scrolling.
Added a new onKeyDown listener which stops page scrolling via `event.preventDefault` as `event.stopPropagation()` was not enough.
Made this functionality optional via a setter and constructor parameter.